packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -15,17 +15,40 @@
 
 import {ImageLoader, Texture as ThreeTexture} from 'three';
 
-import {EmbeddedImage as GLTFEmbeddedImage, ExternalImage as GLTFExternalImage, Image as GLTFImage} from '../../three-components/gltf-instance/gltf-2.0.js';
+import {EmbeddedImage as GLTFEmbeddedImage, ExtensionDictionary, ExternalImage, ExternalImage as GLTFExternalImage, Image as GLTFImage} from '../../three-components/gltf-instance/gltf-2.0.js';
 
 import {Image as ImageInterface} from './api.js';
 import {$correlatedObjects, $onUpdate, $sourceObject, ThreeDOMElement} from './three-dom-element.js';
 
+
+
 const loader = new ImageLoader();
 
 const $threeTextures = Symbol('threeTextures');
 const $uri = Symbol('uri');
 const $bufferViewImages = Symbol('bufferViewImages');
 
+export class GLTFImageDefinition implements ExternalImage {
```

Comment:
Again, I think you can remove all of this and just construct an object directly like `{name: 'null_image', uri: 'null_image'}`.

Reply:
Looks like it's possible, but typescript question, is that good practice long term though? Seems like well defined types could provide some safety and readability.